linera-service/src/proxy.rs

```diff
@@ -45,54 +49,104 @@ pub struct ProxyOptions {
     /// The number of Tokio worker threads to use.
     #[arg(long, env = "LINERA_PROXY_TOKIO_THREADS")]
     tokio_threads: Option<usize>,
+
+    /// Storage configuration for the blockchain history and security states.
```

Comment:
(I find "security states" a bit confusing; but I guess we use that in `server.rs`, too.)

Reply:
Maybe just "chain states"?

Reply:
Ah, and now: "and binary blobs"?